terragraph_planner/los/base_los_validator.py

```diff
@@ -127,18 +127,19 @@ def _get_four_corners_of_rectangle(
         ac and ab are parallel to each other
         ab and dc are parallel to each other
         """
-        if utm_x1 == utm_x2:
-            a = utm_x1 + radius, utm_y1
-            b = utm_x1 - radius, utm_y1
-            c = utm_x2 + radius, utm_y2
-            d = utm_x2 - radius, utm_y2
-        else:
+        # Avoid divide by 0 precision errors by using larger value in denominator
+        if abs(utm_x2 - utm_x1) >= abs(utm_y2 - utm_y1):
             slope = (utm_y2 - utm_y1) / (utm_x2 - utm_x1)
             offset_y = radius / math.sqrt(1 + slope * slope)
             offset_x = offset_y * slope
+        else:
+            slope_inv = (utm_x2 - utm_x1) / (utm_y2 - utm_y1)
+            offset_x = radius / math.sqrt(1 + slope_inv * slope_inv)
```

Comment:
Compared to previous implementation, this will always be positive. In previous implementation, offset_x will be negative if slope is negative.

Reply:
The only difference it would make is that `a` and `c`, `b` and `d` are both switched, which does not matter.